### PR TITLE
Feature/osmt 367 remove quickstart option

### PR DESCRIPTION
**Pull request by schagantiwgu**
Removed the quickstart feature in OSMT.
Deleted the -q option and deleted the "start_osmt_quickstart" method.
Removed instances of "QUICKSTART_ENV_FILE" in the project and deleted quickstart config files

### CODE REVIEW

**Comment by Corpratespaz** [comment]
Please remove these files and any others related to quickstart:
- [ ] https://github.com/wgu-opensource/osmt/blob/develop/docker-compose.quickstart.yml
- [ ] https://github.com/wgu-opensource/osmt/blob/develop/osmt-quickstart.env.example

File: osmt_cli.sh [hunk 1]
```diff
@@ -231,13 +205,12 @@ start_osmt_api_tests() {
...
   remove_osmt_docker_artifacts_for_stack "osmt_dev"
   remove_osmt_docker_artifacts_for_stack "osmt_api_test"
-  remove_osmt_docker_artifacts_for_stack "osmt_quickstart"
 }
 
 cleanup_osmt_docker_artifacts() {
   local prompt_msg; prompt_msg="$(cat <<-EOF
 ${INDENT}Do you want to clean up OSMT-related Docker images and volumes?
-${INDENT}This step will delete data from local OSMT Quickstart and Development configurations.
+${INDENT}This step will delete data from local OSMT and Development configurations.
```
Comment (JohnKallies) [inline osmt_cli.sh:213]: This sentence needs polished

File: osmt_cli.sh [hunk 2]
```diff
@@ -264,11 +237,8 @@ A command line utility to simplify onboarding with OSMT development instances. T
 Usage:
   osmt_cli.sh [accepts a single option]
 
-  -i   Initialize environment files for Quickstart and Development configurations.
+  -i   Initialize environment files for Development configurations.
```
Comment (JohnKallies) [inline osmt_cli.sh:240]: Development and API Test configurations

File: osmt_cli.sh [hunk 3]
```diff
@@ -281,8 +251,7 @@ Usage:
   -r   Start the local Spring app to reindex ElasticSearch.
   -a   Start the local API tests for OSMT. This requires a valid OSMT jar file (from a 'mvn package')
   -m   Import default BLS and O*NET metadata into local Development instance.
-  -c   Surgically clean up OSMT-related Docker images and data volumes. This step will delete data from local OSMT
-       Quickstart and Development configurations. It does not remove the mysql/redis/elasticsearch images, as
+  -c   Surgically clean up OSMT-related Docker images and data volumes. This step will delete data from Development configurations. It does not remove the mysql/redis/elasticsearch images, as
```
Comment (JohnKallies) [inline osmt_cli.sh:254]: Development and API Test configurations

File: .gitignore [hunk 4]
```diff
@@ -45,4 +45,5 @@ osmt*.env
 /test/postman/osmt-auth.environment.json
 /test/postman/token.env
 /test/node_modules
+test/postman/node_modules/
```
Comment (JohnKallies) [inline .gitignore:48]: This change is unrelated. If it is needed, please push it as a separate commit.

Comment (schagantiwgu) [reply .gitignore:48]: It is pushed as a separate commit to this feature branch

File: osmt_cli.sh (hunk 1 above)
Comment (schagantiwgu) [reply osmt_cli.sh]: updated

File: osmt_cli.sh (hunk 3 above)
Comment (schagantiwgu) [reply osmt_cli.sh]: modified the text accordingly

File: osmt_cli.sh (hunk 2 above)
Comment (schagantiwgu) [reply osmt_cli.sh]: modified the text accordingly

File: .gitignore (hunk 4 above)
Comment (JohnKallies) [reply .gitignore:48]: If you remove your local test/postman/node_modules/ directory, it won't be rebuilt. Please remove this line from this PR.

Comment (schagantiwgu) [reply .gitignore]: Updated

**Review by JohnKallies — APPROVED** [review]
FYI @Corpratespaz
